docs/.vuepress/tools/jsdoc-convert/index.mjs

```diff
@@ -1,129 +1,22 @@
-/// npm dependencies
-import fs from 'fs';
-
+/* eslint-disable no-useless-escape */
 /// shared dependencies
 import utils from '../utils.js';
 
 /// internal dependencies
-import configuration from './configuration.mjs';
-import { isJsdocPlugin } from './predictors.mjs';
-import { buildHeaderWriter } from './seo.mjs';
+import configuration from './configuration.js';
```

Comment:
```suggestion
import configuration from './configuration.mjs';
```
On this branch, I see two files with the same content:`configuration.js` and `configuration.mjs`. I guess that we can use `.mjs` format and remove the `.js` file?

Reply:
3613e7f7c28eb7324c403791e50176897d7abbff